settings.gradle

```diff
@@ -141,19 +141,19 @@ dependencyResolutionManagement {
             // `jakartaJpaVersion` comes from the local-build-plugins to allow for command-line overriding of the JPA version to use
             def jpaVersion = version "jpa", "${jakartaJpaVersion}"
 
-            def annotationVersion = version "annotation", "2.1.1"
-            def cdiVersion = version "cdi", "4.0.1"
+            def annotationVersion = version "annotation", "3.0.0"
+            def cdiVersion = version "cdi", "4.1.0"
             def elVersion = version "el", "5.0.0"
             def injectVersion = version "inject", "2.0.1"
-            def interceptorsVersion = version "interceptors", "2.1.0"
-            def dataVersion = version "data", "1.0.0"
-            def jaccVersion = version "jacc", "2.1.0"
-            def jaxbApiVersion = version "jaxbApi", "4.0.0"
-            def jaxbRuntimeVersion = version "jaxbRuntime", "4.0.2"
-            def jsonbApiVersion = version "jsonbApi", "3.0.0"
-            def jsonbRuntimeVersion = version "jsonbRuntime", "3.0.2"
+            def interceptorsVersion = version "interceptors", "2.2.0"
+            def dataVersion = version "data", "1.0.1"
+            def jaccVersion = version "jacc", "3.0.0"
+            def jaxbApiVersion = version "jaxbApi", "4.0.2"
+            def jaxbRuntimeVersion = version "jaxbRuntime", "4.0.5"
+            def jsonbApiVersion = version "jsonbApi", "3.0.1"
+            def jsonbRuntimeVersion = version "jsonbRuntime", "3.0.4"
             def jtaVersion = version "jta", "2.0.1"
-            def validationVersion = version "validation", "3.0.2"
+            def validationVersion = version "validation", "3.1.0"
```

Comment:
hey @gavinking 
I've noticed you've updated the Validation spec version. It probably doesn't matter that much since 3.1/3.0 are identical, but just in case, Hibernate Validator 8 is Jakarta Bean Validation 3.0 and Hibernate Validator 9 is targeting 3.1. There's a beta release of it available. And then if we bump validation to 3.1 we probably should also bump EL to 6.0 as well (at least that's the HV 9 relies on),  it's currently set to `5.0.0`

Reply:
Ah, OK, thanks, I guess I should revert that then.

Reply:
that ... I don't know 😃 
Jakarta validation 3.1 is part of Jakarta 11 so maybe we should keep it and bump the HV version to 9? In ORM HV is used only for tests, right?

Reply:
OK I think I see what you mean: you're saying that HV itself is only used as in testing, even thought it's wrongly classified under `libs` instead of `testLibs`, and se we could use the beta release of H9 there. That makes sense to me.